### DISCUSSION TITLE
[#1113035] Crosshair mode freezes the game and doesn't unfreeze.

### DISCUSSION THREAD

**Discussion by hexagonstudios0** [Mojang/minecraft-editor#141] [Bugs]
If you go into crosshair mode using CTRL + TAB 4+ times, your game will freeze indefinitely.

Video:
[Microsoft Stream](https://bushy70-my.sharepoint.com/:v:/g/personal/fox_bushy70_onmicrosoft_com/EbtrZ4w-Eu1Lm0hwYFLxBsMBG6tdbTVBKEChPCJsZSsRKg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZyIsInJlZmVycmFsQXBwUGxhdGZvcm0iOiJXZWIiLCJyZWZlcnJhbE1vZGUiOiJ2aWV3In19&e=0FdVGv)

**Comment by minecraftchloe** [comment]
Ugh!! This is happening again!! We know about it, digging in :) Workaround for now: 
- When in Tool Mode, use the UI button for now
- When in Crosshair Mode, use `Esc`
